model/task/db.go

```diff
@@ -160,6 +160,32 @@ var (
 		},
 	}
 
+	sortDisplayName = bson.M{
+		"$sort": bson.M{
+			"display_name": 1,
```

Comment:
`DisplayNameKey` assuming this refers to a task display name?

Reply:
This refers to BV display name. Renamed (changed it before requesting review but forgot to actually push it!)